analyser/apkAnalyser.go

```diff
@@ -0,0 +1,77 @@
+package analyser
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+)
+
+//apkAnalyser is an IAnalyser implementation for Android platform
+type apkAnalyser struct {
+	filePath *string
+}
+
+//PrintAppInfo fetches and prints information about the Android app package
+func (analyser apkAnalyser) PrintAppInfo() {
+	appInfo := analyser.getAppInfo()
+	fmt.Printf("Recognised app info:\n "+
+		"\tPackage Name: %+v\n "+
```

Comment:
`%s` would be enough as every field of appInfo is a string.  `%v`: the value in a default format when printing structs, the plus flag (`%+v`) adds field names: https://golang.org/pkg/fmt/#hdr-Printing